entity.js

```diff
@@ -39,15 +39,15 @@ function gentestitem(b, c, d) {
 		b +
 		"</td>" +
 		"<td>" +
-		c.characters +
+		c.characters.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt') +
```

Comment:
Should `'&gt'` be `'&gt;'`?

Reply:
Technically it's the same but for consistency's sake yes.